README.md

```diff
@@ -4,9 +4,13 @@ OSPSuite.ReportingEngine implementation in R
 
 <!-- badges: start -->
 
+  ![](https://img.shields.io/github/downloads/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/latest/total?label=%E2%AD%B3%20Downloads%20latest%20release)

```

Comment:
Initially I wanted to test and include arbitrary svg images in the badge. Hence the long encoded logo link.
But actually it is much simpler to include directly the download symbol as a character.

Reply:
Nice!